test/testHelpers.ts

```diff
@@ -1,7 +1,22 @@
 import nock from "nock";
 import { Onfido, Region } from "onfido-node";
 
-export const onfido = new Onfido({ region: Region.EU, apiToken: "api_token" });
+export const onfido = new Onfido({ region: Region.EU, apiToken: process.env.ONFIDO_API_TOKEN || "api_token" });
 
 export const createNock = (): nock.Scope =>
   nock("https://api.eu.onfido.com/v3.4");
+
+export function getExpectedObject( exampleObject: any ) {
```

Comment:
Should we extend this getExpectedObject to match all fields returned in the API?

Reply:
I'm not sure if I've got it right. Aim of this function is just to adapt object provided as example to the mock library to allow wider range of values for fields that are not predictable when using the mocked API.